### DISCUSSION TITLE
Improving Discoverability

### DISCUSSION THREAD

**Discussion by austin4195** [ServerListPlus/issues#18] [Ideas]
One thing that server lists struggle with is improving discoverability. You typically have your paid slots and then the highest votes (or whatever metric) displayed but what if there was also a slot near the top of the page that was random?

Using my other idea #17, verified servers would be randomly displayed to improve visibility. Servers near the top of metrics and or premium slots wouldn’t be eligible but it would give the little guys (like myself) a chance for a brief moment.

**Comment by Brend-Smits** [comment]
That's a good idea. We could also show a random active server in between the 'premium' section (not entirely sure if there will be any but have to see how it goes).
I have to give this more thought but this is definitely something I want to look into adding.

**Reply by austin4195** [reply]
The verification just made it so any random server may not be displayed but that it required some effort to get there if that makes sense.

One approach to do it (and not too sure how feasible it would be) would be that each time the webpage is served it plucks one of those verified servers to display. This could mean that two devices on the same network would still show a different server.

Otherwise, could consider it changing on a time interval.
